.github/workflows/release.yml

```diff
@@ -0,0 +1,38 @@
+name: release
+on:
+  create:
+    tags:
+      - v*
+
+jobs:
+  release:
+    if: ${{ github.repository_owner == 'helm'}}
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout source code
+        uses: actions/checkout@755da8c3cf115ac066823e79a1e1788f8940201b # pin@v3.2.0
+
+      - name: Setup Go
+        uses: actions/setup-go@6edd4406fa81c3da01a34fa6f6343087c207a568 # pin@3.5.0
+        with:
+          go-version: '1.18'
+
+      - name: Set Version
+        id: set-version
+        run: |
+          echo "VERSION=$(echo ${GITHUB_REF/refs\/tags\//})" >> $GITHUB_OUTPUT
```

Comment:
Since this is already filtered for tags, is there a reason we aren't using the `GITHUB_REF_NAME` variable?

Reply:
Fixed